src/fontem.c

```diff
@@ -379,7 +379,7 @@ void store_glyph(FT_Face *face, FT_GlyphSlotRec *glyph,
 
 	// Append to the post
 	char *str = malloc(len + 100);
-	snprintf(str, len + 100, "\t[%u] = &%s,  /* '%s' */\n", ch, gname, mb(ch));
```

Comment:
Is it worth adding the hex value of `ch` inside the comment block? Purely for a diagnostic aid.

Reply:
Done.